src/kakarot/memory.cairo

```diff
@@ -418,4 +418,34 @@ namespace Memory {
         }
         return (new_memory=self, cost=0);
     }
+
+    // @notice Expand memory if necessary then load 32 bytes from it at given offset.
+    // @param self - The pointer to the memory.
+    // @param offset - The number of bytes to add.
+    // @return The new pointer to the memory.
+    // @return The loaded Uint256 from memory.
+    // @return The gas cost of this expansion.
+    func expand_and_load{range_check_ptr}(self: model.Memory*, offset: felt) -> (
```

Comment:
I wonder too:)! It's simply to give freedom to have access to a Memory.load function that does not expand memory.

Otherwise:
Rename `Memory.load` and `Memory.load_n` to `Memory._load` and `Memory._load_n`. They would be "internal" so to speak.
Make Memory.load and Memory.load_n be the outward facing, memory expanding load functions.

Reply:
> does

Sound good to me